apps/web/src/content/blog/en/transform-lovable-dev-app-to-mobile-with-capacitor.md

```
@@ -125,254 +144,241 @@ Now let's get your Lovable.dev project into Cursor:
-   Install Homebrew, Node.js and npm on my system, then install dependencies and run the dev server
-   ```
-
-The AI will automatically:
-- Detect your operating system
-- Install Homebrew (on macOS)
-- Install Node.js and npm
-- Navigate to your project directory
-- Run `npm install` to install dependencies
-- Start your development server with `npm run dev`
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)